src/Commands/Remote/SSHBaseCommand.php

```diff
@@ -247,13 +273,18 @@ private function getOutputCallback()
      * Returns the command-line args.
      *
      * @param string[] $command_args
+     * @param array $env_vars
      *
-     * @return string
+     * @return array
+     *   Elements are as follows:
+     *     string command_line The command line string
+     *     array env_vars The environment variables
      */
-    private function getCommandLine($command_args)
+    private function getCommandLine($command_args, $env_vars)
     {
-        array_unshift($command_args, $this->command);
-        return implode(" ", $this->escapeArguments($command_args));
+        $command_line = implode(" ", $this->escapeArguments(array_merge([$this->command], $command_args)));
```

Comment:
I would convert the `$env_vars` array `$k => $v` into a series of `$k=$v` strings at the beginning of `$command_line`